HW7.py

```diff
@@ -0,0 +1,21 @@
+"""
+Homework 7. If-elif-else.
+
+Given values w,x,y,z
+write if-elif-else statement that will search minimum value
+and print smth aka "'y' is minimum value'
+where 'y' is variable name (not value)
+advice use python debugger and different values to check your algorithm
+optionally you can check your algorithm somehow with assert statements
+"""
+
+w, x, y, z = -1, -1, -1, 0
+
+if (w <= x) and (w <= y) and (w <= z):
+    print("'w' is minimum value")
```

Comment:
Such text better to place as template 
`template = "'{}' is minimum value"`
and then do just format 
`print(template.format('w'))`
this will reduce duplicate and template could be changed in single place